traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -1432,7 +1432,7 @@ sub get_qstring_ignore_remap {
 	my $ats_major_version = shift;
 
 	if ($ats_major_version >= 6) {
-		return " \@plugin=cachekey.so \@pparam=--separator= \@pparam=--remove-all-params=true \@pparam=--remove-path=true \@pparam=--capture-prefix-uri=/http:\\/\\/([^?]*)/http:\\/\\/\$1/";
+		return " \@plugin=cachekey.so \@pparam=--separator= \@pparam=--remove-all-params=true \@pparam=--remove-path=true \@pparam=--capture-prefix-uri=/https?:\\/\\/([^?]*)/http:\\/\\/\$1/";
```

Comment:
I think this isn't quite right. An origin may legitimately serve different content on its http and https ports, but this conflates the two into a single cachekey. The scheme should be part of the key. Might `/^([^?]*)/$1/` work better, to simply include the scheme, which we know won't have a `?` in it?

Reply:
Ill compare to what we had before. I know we previously had cachekeys that took all http/https and sent it to the single http key and I thought the ignore was similar

Reply:
Yes looks like that one might work, however ATS is doing something funky with https. For http the URI its using is just fine, however on an https you end up with something like:
< X-Cache-Key: localhost:443/

Its stripping the https and bringing in the port